dali/pipeline/operators/decoder/nvjpeg_decoder.cu

```diff
@@ -37,7 +37,13 @@ is encountered and internal buffer needs to be reallocated to decode it.)code",
       R"code(Padding for nvJPEG's host memory allocations in bytes.
 This parameter helps to avoid reallocation in nvJPEG whenever a bigger image
 is encountered and internal buffer needs to be reallocated to decode it.)code",
-      16*1024*1024);
+      16*1024*1024)
+  .AddOptionalArg("cache_size",
+      R"code(Total size of the decoder cache in bytes. When provided, decoded
+images bigged than `image_size_threshold` will be cached in memory.)code",
```

Comment:
What is `image_size_threshold` ? How do we set it?

Reply:
it's `cache_threshold`. Will fix